contributing.md

```diff
@@ -113,12 +113,15 @@ _(note that running mypy and black with no errors is required before code will b
 > You can only build an executable for the platform you are running pyinstaller on. The spec for each platform is stored in tabcmd-*platform*.spec and the exact build commands for each platform can be checked in [our packaging script](.github/workflows//package.yml).
 
 e.g for Windows
-> pyinstaller tabcmd-windows.spec --clean --noconfirm --distpath ./dist/windows
+> pyinstaller tabcmd-windows.spec --clean --distpath ./dist/windows
 
-produces dist/tabcmd.exe
+produces dist/windows/tabcmd.exe
```

Comment:
Were the backslashes causing problems? Did they need to be escaped or something?

Reply:
No, you suggested I change them?

Reply:
Yeah, maybe the comment got lost on a previous revision. https://github.com/tableau/tabcmd/pull/302#discussion_r1684926532